src/engine/runtime.js

```diff
@@ -838,7 +864,7 @@ class Runtime extends EventEmitter {
     /**
      * Enqueue a script that when finished will update the monitor for the block.
      * @param {!string} topBlockId ID of block that starts the script.
-     * @param {?string} optTarget target ID for target to run script on. If not supplied, uses editing target.
+     * @param {?string} optTarget target Target to run script on. If not supplied, uses editing target.
```

Comment:
If this really is taking a `Target` now, please update `{?string}` => `{?Target}`

Reply:
Good catches, thanks!